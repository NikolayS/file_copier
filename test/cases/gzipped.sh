#!/bin/bash

# to run this scenario, COPIERSERVICE should be defined (`export COPIERSERVICE=https://your.copier.host.name`)

http GET "$COPIERSERVICE/?wh=1&md5=1&src=https://www.sendwithus.com/assets/img/email_guide_img/swu-guide.jpg?rev=201511231449" -h

