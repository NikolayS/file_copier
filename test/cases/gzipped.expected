HTTP/1.1 200 OK
X-File-Copier-Md5: 437e8f6209872d5a1e69ebfb9e18ab34
X-File-Copier-Img-Width: 1000
X-File-Copier-Img-Height: 500
X-File-Copier-Size: 157024
X-Location: /data/8f/ee/d8/47/8feed847a5462ab554e386d244ff15a35100faff24e34024e7a5bdd4756e70d5.jpg
Access-Control-Expose-Headers: X-File-Copier-Size, X-Location, X-File-Copier-Img-Height, X-File-Copier-Img-Width, X-File-Copier-Md5

